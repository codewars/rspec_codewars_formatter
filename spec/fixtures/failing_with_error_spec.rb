RSpec.describe "Failing with Error" do
  def foo(a, b)
    a / b
  end

  it "errors" do
    expect(foo(1, 0)).to eq(1)
  end
end
