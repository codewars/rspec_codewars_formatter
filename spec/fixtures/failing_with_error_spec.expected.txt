
<DESCRIBE::>Failing with Error

<IT::>errors

<ERROR::>./spec/fixtures/failing_with_error_spec.rb:3:in `/': divided by 0 (ZeroDivisionError)

<LOG::-Backtrace>./spec/fixtures/failing_with_error_spec.rb:3:in `foo'<:LF:>./spec/fixtures/failing_with_error_spec.rb:7:in `block (2 levels) in <top (required)>'

<COMPLETEDIN::>0.64

<COMPLETEDIN::>0.8554
